platzky_redirections/plugin.py

```
@@ -0,0 +1,99 @@
+
+    Raises:
+        ValueError: If URLs are malformed
+    """
+
+    def validate_url(url: str) -> bool:
+        """Check that a URL starts with '/' or 'http'."""
+        return url.startswith(("/", "http"))
+
+    invalid_urls = [url for url in config.keys() | config.values() if not validate_url(url)]
+    if invalid_urls:
+        raise ValueError(f"Invalid URLs found: {invalid_urls}")
```

Ruff (0.15.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)